### PR TITLE
Version getters

### PR DESCRIPTION
**Pull request by Liquidsoul**
This is an implementation for [#501](https://github.com/KrauseFx/fastlane/issues/501).
It is mostly a copy/paste adaptation of [increment_build_number](https://github.com/KrauseFx/fastlane/blob/master/lib/fastlane/actions/increment_build_number.rb) and [increment_version_number](https://github.com/KrauseFx/fastlane/blob/master/lib/fastlane/actions/increment_version_number.rb) to output the same variables.
Maybe this can be factorized a bit but I don't know how this can be done here.

First Ruby code contribution, so any feedback welcomed :smiley:

### CODE REVIEW

File: lib/fastlane/actions/get_build_number.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      end
+
+      def self.output
+        [
+          ['BUILD_NUMBER', 'The build number']
+        ]
+      end
+
+      def self.authors
+        # So no one will ever forget your contribution to fastlane :) You are awesome btw!
```
Comment (KrauseFx) [inline lib/fastlane/actions/get_build_number.rb]: You can remove this line :+1:

File: lib/fastlane/actions/get_build_number.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+          command = [
+            command_prefix,
+            'agvtool',
+            'what-version',
+            '-terse'
+          ].join(' ')
+
+          if Helper.test?
+            Actions.lane_context[SharedValues::BUILD_NUMBER] = command
```
Comment (KrauseFx) [inline lib/fastlane/actions/get_build_number.rb]: You can use this line to test this integration

**Comment by KrauseFx** [comment]
Looks good, thanks for the pull request. Could you also add some tests to the newly added actions?

**Comment by Liquidsoul** [comment]
I do not fully understand how this works yet :wink: 
So, I need some hints on how to add tests.
Where should I look ?

**Comment by Liquidsoul** [comment]
@KrauseFx I may have found how to add tests :smiley:

**Comment by KrauseFx** [comment]
Looks great, thanks for the pull request @Liquidsoul :+1:

**Pull request merged**